### PR TITLE
[13.0] session_db

### PR DESCRIPTION
**Pull request by sbidoul**
Same as #2441, for 13.

### CODE REVIEW

**Comment by sbidoul** [comment]
This one has been used successfully in production since a couple of months now.

**Comment by sbidoul** [comment]
This one has been used successfully in production since a couple of months now.

File: session_db/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Store sessions in a database instead of the filesystem.
```
Comment (legalsylvain) [inline session_db/readme/DESCRIPTION.rst]: If possible, could you mention the interest to store sessions in db instead of filesytem ?

Comment (sbidoul) [reply session_db/readme/DESCRIPTION.rst]: @legalsylvain I added this to the README: *This simplifies the configuration of horizontally scalable deployments, by avoiding the need for a distributed filesystem to store the Odoo sessions.*

**Comment by sbidoul** [comment]
I'm going ahead and merging this. I'll remain available in case any issue show up.

/ocabot merge patch

**Comment by sbidoul** [comment]
I improved handling of postgres errors. cc/ @adrienpeiffer

**Review by adrienpeiffer — APPROVED** [review]
Code review and (deep) functional testing

**Pull request merged**